### PR TITLE
fix: [CDS-46194]: Export GitOps Service version in UI

### PR DESCRIPTION
**Pull request by cjlee01**
### Summary

Export GitOps Service version in UI and show in list 

#### Screenshots

NA

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by Sagar-Rajput7 — APPROVED** [review]

**Review by Tamaraius — APPROVED** [review]

**Review by chetanvekariya-harness — DISMISSED** [review]

File: src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx [hunk 1]
```diff
@@ -129,6 +134,9 @@ const ServiceVersions = () => {
                 if (response.value.version) {
                   // for NGUI
                   serviceRow['version'] = response.value.version
+                } else if (response.value.Version) {
+                  // for Gitops
+                  serviceRow['version'] = response.value.Version
```
Comment (ishantmehtaharness) [inline src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx:139]: We have 2 versions here
response.value.version
And 
response.value.Version

Do we need these 2 separate ?

**Review by ishantmehtaharness — COMMENTED** [review]
Comment

File: src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx (hunk 1 above)
Comment (mteodor) [reply src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx:139]: I'll correct service so that we dont have this special case

Comment (ishantmehtaharness) [reply src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx:139]: Thank You @mteodor

Comment (cjlee01) [reply src/modules/33-auth-settings/pages/AccountOverview/views/ServiceVersions.tsx:139]: removed this case cause not needed anymore @ishantmehtaharness 
fixed be in gitops

**Review by devinder-harness — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request merged**